### ISSUE TITLE
A11y_Azure Tools Storage Explorer _Get Shared Access Signature_ ScreenReader: Voiceover is not announcing the Complete Information in Dequeue message dialog box.

### ISSUE THREAD

**Issue by Latha2564** [Microsoft/AzureStorageExplorer#4989]
### GitHub Tags:
#A11y_Azure Tools Storage Explorer; #A11ySev3; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Azure Tools Storage Explorer; #MAC; #BM-AzureToolsStorageExplorer-MAC-Sep2021; #WCAG1.3.1; #Info and Relationships; #AINotInScope;#Voiceover;#A11yMAS;

### Environment Details:​
Application Name: Azure Tools Storage Explorer
Standard Violated: Info and Relationships

### Repro Steps:​
1.Enable Voice over.
2.Open Azure Tools Storage Explorer.
3.Tab till any existing item in a tree.
4.Tab till "Storage Accounts" and press Enter.
5.Tab till any existing account under Storage account and press enter.
6.Tab till "Queues" and press enter.
7.Tab till an existing queue and press enter.
8. Tab till "Dequeue message" button in Queue toolbar and press enter.
9. verify whether the Voice over is reading the complete information in the appeared dialog box or not.

### Actual Result:
Voiceover is not announcing the Complete Information in Deque message dialog box.
Note:
Similar issue is observed in "Clear Queue" scenario  also.

### Expected Result: ​
Voiceover should announce  the Complete Information in Deque message dialog box.

### User Impact:
Screen reader users will find difficult in finding the particular dialog box if the Voiceover is not announcing the Complete Information in Deque message dialog box.

### Attachments:
![Voice over is not announcing all information in Dialog box](https://user-images.githubusercontent.com/89508906/135092894-d8ef866e-bada-4df6-8a90-a0586f73e684.gif)

**Comment by MRayermannMSFT** [comment]
Should ask accessibility office hours about this. Initially we focus the OK button so that is read. User can use voice over keys to navigate to the text and have it read though. Unsure if reading the whole dialog is the right thing to do or not.
